OpenOversight/app/templates/officer.html

```diff
@@ -58,112 +58,127 @@ <h3>General Information
-                        <p><span style="color: red;">[{{ error }}]</span></p>
-                    {% endfor %}
-              </td>
-            </tr>
+          <tr>
+            <td><b>New badge number:</b></td>
+            <td>{{ form.star_no }}
+                  {% for error in form.star_no.errors %}
+                      <p><span style="color: red;">[{{ error }}]</span></p>
+                  {% endfor %}
+            </td>
+          </tr>
 
-            <tr>
-              <td><b>New rank:</b></td>
-              <td>{{ form.rank }}
-                    {% for error in form.rank.errors %}
-                        <p><span style="color: red;">[{{ error }}]</span></p>
-                    {% endfor %}
-              </td>
-            </tr>
+          <tr>
+            <td><b>New rank:</b></td>
+            <td>{{ form.rank }}
+                  {% for error in form.rank.errors %}
+                      <p><span style="color: red;">[{{ error }}]</span></p>
+                  {% endfor %}
+            </td>
+          </tr>
 
-            <tr>
-              <td><b>New unit:</b></td>
-              <td>{{ form.unit }}
-                    {% for error in form.unit.errors %}
-                        <p><span style="color: red;">[{{ error }}]</span></p>
-                    {% endfor %}
-              </td>
-            </tr>
+          <tr>
+            <td><b>New unit:</b></td>
+            <td>{{ form.unit }}
+                  {% for error in form.unit.errors %}
+                      <p><span style="color: red;">[{{ error }}]</span></p>
+                  {% endfor %}
+            </td>
+          </tr>
 
-            <tr>
-              <td><b>Start date of new assignment:</b></td>
-              <td>{{ form.star_date }}
-                    {% for error in form.star_date.errors %}
-                        <p><span style="color: red;">[{{ error }}]</span></p>
-                    {% endfor %}
-              </td>
-            </tr>
+          <tr>
+            <td><b>Start date of new assignment:</b></td>
+            <td>{{ form.star_date }}
+                  {% for error in form.star_date.errors %}
+                      <p><span style="color: red;">[{{ error }}]</span></p>
+                  {% endfor %}
+            </td>
+          </tr>
 
-            <tr>
-              <td>
-                <input type="submit" value="Add Assignment" name="add-assignment" class="btn-sm btn-primary"/>
-              </td>
-            </tr>
+          <tr>
+            <td>
+              <input type="submit" value="Add Assignment" name="add-assignment" class="btn-sm btn-primary"/>
+            </td>
+          </tr>
 
-            </form>
-            </tbody>
-          </table>
-          {% endif %}
-      </div>
-    </div>
-  </div>
-</div>
+          </form>
+        </tbody>
+      </table>
+      {% endif %}
+    </div> {# end col #}
+    <div class='col-sm-12 col-md-6'>
+      <h3>Incidents</h3>
```

Comment:
This profile page is looking sweet 😎 . We should also have a link to the edit incident page like we do on the general information page next to the incident:

<img width="357" alt="screen shot 2018-05-14 at 9 33 38 pm" src="https://user-images.githubusercontent.com/7832803/40036806-7d6c5408-57be-11e8-8859-36efbc241046.png">

And an "Add incident" button that takes us to the new incident page: 

<img width="182" alt="screen shot 2018-05-14 at 9 34 17 pm" src="https://user-images.githubusercontent.com/7832803/40036824-98ff45a4-57be-11e8-87f3-4f888166c3c2.png">